Detectors/MUON/MCH/Raw/Common/test/testSampaCluster.cxx

```diff
@@ -26,36 +26,37 @@ BOOST_AUTO_TEST_SUITE(o2_mch_raw)
 BOOST_AUTO_TEST_SUITE(sampacluster)
 
 uint16_t defaultTimestamp{0x3FF};
+uint32_t defaultBunchCrossing{0xFFFF};
```

Comment:
that's not a valid default is BC is 20 bits, right ? 
should also add a test to actually ensure the SampaCluster is throwing if an attempt is made to construct it with more than 20-bits BC.

Reply:
I fixed it. I still need to see how to add a test for the BC range.